README.md

```diff
@@ -1,110 +1,157 @@
 # Inspirehep
 
-### Installation
+## Pre requirements
 
-#### poetry (optional)
+### poetry
 
 install `poetry` https://poetry.eustace.io/docs/
 
 ```bash
 $ curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python -
 ```
 
-### How to run
+### pre-commit
 
-First you will need a working kubernetes cluster.
-On Mac OS, you can use [docker-desktop](https://www.docker.com/products/docker-desktop) but due to [docker/for-mac/issue#2990](https://github.com/docker/for-mac/issues/2990), [minikube](https://minikube.sigs.k8s.io/docs/start/macos/) is recommended also for Mac.
-On Linux, you can use [minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).
-Once you have access to your local kubernetes cluster, following the instructions to install [Tilt](https://docs.tilt.dev/install.html).
-Make sure that you have set up ssh for gitlab (https://docs.gitlab.com/ee/ssh/)
-Make sure you have `kustomize` version at least `3`
-You can then start your environment using
+install `pre-commit` https://pre-commit.com/
```

Comment:
On fresh Ubuntu install before running this user should install python3-distutils`

Reply:
Also after installing pre-commit user should add `/home/$USER/bin` to his `$PATH`
`echo  'export PATH=$PATH:/home/$USER/bin' >> ~/.bashrc`

Reply:
isn't this part of the poetry installation guide? Should we added to our readme?

Reply:
Adding /home/$USER/bin to PATH is not literally mentioned on poetry installation guide. there is only: 
```
The installer installs the poetry tool to Poetry's bin directory. On Unix it is located at $HOME/.poetry/bin and on Windows at %USERPROFILE%\.poetry\bin.

This directory will be in your $PATH environment variable, which means you can run them from the shell without further configuration.
```
But it didn't add it automatically to my PATH on Ubuntu LTS
Python 3 and python3 distutils is not mentioned on poetry guide at all.

Reply:
but what about the ones run zsh?, I don't want to be that specific that's why we have the link to poetry

Reply:
As you prefer. I'm just putting my thoughts here ;)

Reply:
👍 let me know if you have any other comments